CONFIGURING-DETECTORS.md

```diff
@@ -0,0 +1,103 @@
+
+#### Predefined edge inference configs
+
+There are a few predefined configs in the configuration file. These make use of the available edge inference parameters to achieve different kinds of behavior. If you're just getting started with setting up the edge endpoint, we recommend choosing from these based on which best fits your desired behavior!
+
+##### `default`
+```
+default: # Return the edge model's prediction if sufficiently confident; otherwise, escalate to the cloud.
+    enabled: true
+    always_return_edge_prediction: false
+    disable_cloud_escalation: false
+```
+This is the default behavior for a detector on the edge and is likely what you'll want to use most of the time, unless you have a specific reason to use a different configuration.
+
+##### `edge-answers-with-escalation`
+```
+edge-answers-with-escalation: # Always return the edge model's predictions, but still escalate to cloud if unconfident.
+    enabled: true
+    always_return_edge_prediction: true
+    disable_cloud_escalation: false
+    min_time_between_escalations: 2.0
+```
+Use this config if: you want all answers to come from the edge model to ensure quick response times. This will happen regardless of the answers' confidence. However, your unconfident queries will be escalated to the cloud. This allows Groundlight (or you!) to provide labels on your detector so your model can improve over time.
+
+##### `no-cloud`
+```
+no-cloud: # Always return the edge model's prediction and never escalate to the cloud.
+    enabled: true
+    always_return_edge_prediction: true
+    disable_cloud_escalation: true
+```
+Use this config if: you always want the fastest answers from the edge model and don't want any image queries to get escalated to the cloud. This might be useful under circumstances such as limited network bandwith. Be careful using this option: if unconfident queries aren't escalated to the cloud, the model won't be able to receive new labels and improve. 
+
+##### `disabled`
+```
+disabled:
+    enabled: false
+```
+Use this config if: you don't want the edge endpoint to accept image queries for this detector.
+
+### `detectors`
+
+This section is where you define your detectors, along with the edge inference config to use for each of them. The structure looks like:
+```
+detectors:
+    - detector_id: "det_abc"
+        edge_inference_config: "default"
+    - detector_id: "det_xyz"
+        edge_inference_config: "no-cloud"
+```
+Add a new entry for each detector that you want to configure. Remember that inference configs can be applied to as many detectors as you'd like, so if you want multiple detectors to have the same configuration, just assign them the same edge inference config. Make sure you don't have multiple entries for the same detector - in this case, the edge endpoint will error when starting up.
```

Comment:
Maybe we should also explicitly state that users can define new configs that fit their desired behavior?

Reply:
Good idea!